AlphaWallet/Transactions/Coordinators/TicketsCoordinator.swift

```diff
@@ -0,0 +1,611 @@
+//  TicketsCoordinator.swift
```

Comment:
`AlphaWallet/Transactions/Coordinators/TicketsCoordinator.swift` is a leftover from previous rename/refactoring. We can remove it.